src/tests.rs

```diff
@@ -849,3 +849,43 @@ fn range_and_reduction() -> TestResult {
 fn precedence_of_or_groups() -> TestResult {
     run_test(r#"4 mod 3 == 0 || 5 mod 5 == 0"#, "true")
 }
+
+#[test]
+fn str_case_camel_cell_path() -> TestResult {
+    run_test(
+        r#"([[lang, gems]; [nu_test, 100]]).lang | get 0 | str camel-case"#,
+        "nuTest",
+    )
+}
+
+#[test]
+fn str_case_kebab_cell_path() -> TestResult {
+    run_test(
+        r#"([[lang, gems]; [nuTest, 100]]).lang | get 0 | str kebab-case"#,
+        "nu-test",
+    )
+}
+
+#[test]
+fn str_case_pascal_cell_path() -> TestResult {
+    run_test(
+        r#"([[lang, gems]; [nu_test, 100]]).lang | get 0 | str pascal-case"#,
+        "NuTest",
+    )
+}
+
+#[test]
+fn str_case_snake_cell_path() -> TestResult {
+    run_test(
+        r#"([[lang, gems]; [nuTest, 100]]).lang | get 0 | str snake-case"#,
+        "nu_test",
+    )
+}
+
+#[test]
+fn str_case_screaming_snake_cell_path() -> TestResult {
+    run_test(
+        r#"([[lang, gems]; [nuTest, 100]]).lang | get 0 | str screaming-snake-case"#,
```

Comment:
Thanks for adding some tests here. Can we change some of these to take in a column path? These are using the default but not taking in an argument that gives the column to update.

Reply:
Oh, I see, I was not sure how to do this. Thanks for checking and clarifying it!

Reply:
Fixed it!